react/src/components/DeleteMapModal/DeleteMapModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import { Button, SectionMessage } from '../../core-components';
+import styles from './DeleteMapModal.module.css';
+import { Project } from '../../types';
+import { useDeleteProject } from '../../hooks/projects/';
+
+type DeleteMapModalProps = {
+  isOpen: boolean;
+  toggle: () => void;
+  projectId?: number;
+  project?: Project;
+};
+
+const DeleteMapModal = ({
+  isOpen,
+  toggle: parentToggle,
+  projectId,
+  project,
+}: DeleteMapModalProps) => {
+  const [errorMessage, setErrorMessage] = useState('');
+  const { mutate: deleteProject, isLoading: isDeletingProject } =
```

Comment:
```suggestion
  const { mutate: deleteProject, isLoading: isDeletingProject, isError} =
```

suggestions: could `isError` be used.  Then you could simplify things and remove `const [errorMessage, setErrorMessage] = useState('');`